src/installer/__main__.py

```diff
@@ -0,0 +1,82 @@
+"""Installer CLI."""
+
+import argparse
+import distutils.dist
+import sys
+import sysconfig
+from typing import Dict, Optional, Sequence
+
+import installer
+import installer.destinations
+import installer.sources
+import installer.utils
+
+
+def main_parser() -> argparse.ArgumentParser:
+    """Construct the main parser."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("wheel", type=str, help="wheel file to install")
+    parser.add_argument(
+        "--destdir",
+        "-d",
+        metavar="/",
+        type=str,
+        default="/",
+        help="destination directory (prefix to prepend to each file)",
+    )
+    parser.add_argument(
+        "--optimize",
+        "-o",
+        nargs="*",
+        metavar="level",
+        type=int,
+        default=(0, 1),
+        help="generate bytecode for the specified optimization level(s) (default=0, 1)",
```

Comment:
is this able to be turned off entirely if desired?

Reply:
Should be. That's being discussed further down in the thread.